modules/keepnick.cpp

```diff
@@ -174,7 +174,7 @@ class CKeepNickMod : public CModule {
                 return HALT;
             }
 
-            PutModule("Unable to obtain nick: " + numeric.GetParam(2) + ": " + numeric.GetParam(3));
+            PutModule("Unable to obtain nick: " + numeric.GetParam(1) + ": " + numeric.GetParam(2));
```

Comment:
"message" is just a last parameter, there's nothing special about number of parameters.
`GetParam(2)` will be empty for 447, because `GetParam(1)` has leading colon and therefore is the last parameter.
Is `goodstuff` the current nick?